server.js

```diff
@@ -5590,6 +5590,57 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Swagger Validator integration.
+camp.route(/^\/swagger\/(v)\/(2\.0)\/(https?)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var type = match[1];        // e.g. `v` for validate
+  var specVer = match[2];     // e.g. `2.0` for OpenAPI 2.0
+  var scheme = match[3];      // e.g. `https`
+  var swaggerUrl = match[4];  // e.g. `api.example.com/swagger.yaml`
+  var format = match[5];
+
+  var badgeData = getBadgeData('swagger', data);
+
+  var urlParam = encodeURIComponent(scheme + '://' + swaggerUrl);
+  var url = 'http://online.swagger.io/validator/debug?url=' + urlParam;
```

Comment:
Hehe, I would say, it's probably not worth it.

Strange, the certificate error didn't pop up in my browser.